backend/build.gradle

```diff
@@ -26,6 +26,8 @@ kotlin {
     jvmToolchain(21)
     // Added to dismiss warnings, see: https://github.com/loculus-project/loculus/issues/4529
     compilerOptions.freeCompilerArgs.add("-Xannotation-default-target=param-property")
+    // required since kotlinx-datetime:0.7.0
```

Comment:
This comment is not really useful in the future and I think that it is easy enough to find out what it does. I'd also suspect that once the API is stable and the flag is not needed, the compiler will show a corresponding warning.

```suggestion
```